docker/docker-compose.yml

```diff
@@ -1,17 +1,17 @@
 ﻿version: "3.7"
 services:
   mongodb:
-    image: mongo:4.2.3
+    image: mongo:4.2
```

Comment:
```suggestion
    image: mongo:4.4
```
i'm out of the loop, but from catching up on slack it seems like the servers are on 4.4 now, so maybe this could be too?

Reply:
the production and dev servers are on 4.4 indeed

Reply:
I've applied the change. Thanks @svensven!